src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+const MALE_GENDER = 'm';
+
+export const Person = (props) => {
+  const { name, age, sex, isMarried, partnerName } = props.person;
+  let partner = `I am not married`;
```

Comment:
It's not clear why not marriage info string is stored in `partner` variable
Check please variable name